site/_resources/release-process.md

```diff
@@ -3,136 +3,269 @@ title: Contour Release Process
-Create a release branch locally, like so
+1. Push the tag to `github.com/projectcontour/contour`:
 
-```sh
-$ git checkout -b release-1.7
-```
+    ```bash
+    git push ${CONTOUR_UPSTREAM_REMOTE_NAME} ${CONTOUR_RELEASE_VERSION}
+    ```
 
-Push the release branch to the upstream repository:
+### Release the operator
 
-```sh
-$ git push --set-upstream <YOUR UPSTREAM REMOTE NAME> release-1.7
-```
+1. Check out `main`, ensure it's up to date, and ensure you have a clean working directory.
+1. Create a local release branch:
 
-If you are doing a patch release on an existing branch, skip this step and just checkout the branch instead.
+    ```bash
+    git checkout -b release-${CONTOUR_RELEASE_VERSION_MAJOR}.${CONTOUR_RELEASE_VERSION_MINOR}
+    ```
 
-This branch is used for all the almost-done release types, rc and final.
-Each one of these release types is just a different tag on the release branch.
+1. Push the branch to `github.com/projectcontour/contour-operator`:
 
-## Final release
+    ```bash
+    git push --set-upstream ${CONTOUR_OPERATOR_UPSTREAM_REMOTE_NAME} release-${CONTOUR_RELEASE_VERSION_MAJOR}.${CONTOUR_RELEASE_VERSION_MINOR}
+    ```
 
-These two steps can be automatically performed by the `hack/release/make-release-tag.sh` tool.
-For example:
+1. Update the deployment YAML and create a local tag:
 
-```sh
-$ ./hack/release/make-release-tag.sh v1.2.1 v1.3.1
-```
+    ```bash
+    ./hack/release/make-release-tag.sh main $CONTOUR_RELEASE_VERSION
+    ```
 
-### Updating the example YAMLs
+1. Push the branch to `github.com/projectcontour/contour-operator`:
 
-Your final job before doing the last release is to ensure that all the YAMLs in `examples/contour/` are updated.
-The Docker tag should be updated from the previous stable release to this new one.
+    ```bash
+    git push ${CONTOUR_OPERATOR_UPSTREAM_REMOTE_NAME} release-${CONTOUR_RELEASE_VERSION_MAJOR}.${CONTOUR_RELEASE_VERSION_MINOR}
+    ```
 
-### Tag release from release branch
+1. Push the tag to `github.com/projectcontour/contour-operator`:
 
-Tag the head of your release branch with the release tag, and push
+    ```bash
+    git push ${CONTOUR_OPERATOR_UPSTREAM_REMOTE_NAME} ${CONTOUR_RELEASE_VERSION}
+    ```
 
-```sh
-$ git tag -a v1.2.0 -m 'contour 1.2.0'
-$ git push --tags
-```
+### Update quickstart YAML redirects
+
+1. Check out `main`, ensure it's up to date, and ensure you have a clean working directory.
+1. Create a new local feature branch from `main`.
+1. Update `netlify.toml` to redirect quickstart links to the new release branch.
+1. Commit all changes, push the branch, and PR it into `main`.
+
+### Do the Github release and write release notes
+
+Now you have a tag pushed to Github, go to the release tab on github, select the tag and write up your release notes.
+
+_Note: Filter on the Github label "release note" and Github milestone which should include any PRs which should be called out in the release notes._ 
+
+### Toot your horn
```

Comment:
Should we mention the blog post for the release here or is that action item captured elsewhere?

Reply:
Yep good catch, I'll add that.